### DISCUSSION TITLE
Creazione dei vari teams

### DISCUSSION THREAD

**Discussion by guizzo** [Il-Libro-Open-Source/book#132] [Idee]
Ciao bella gente 👋 

Che ne dite di creare dei teams qui su Github dedicati ai vari [Working Groups](https://github.com/Il-Libro-Open-Source/governance/blob/main/WG.md#working-group-attivi) attivi?

Questo credo possa tornarci comodo quando andiamo a lavorare sulle PR, ad esempio: anziché taggare singolarmente uno o più membri del Drafting Group per chiedere una review sulla forma e sulla grammatica utilizzata, ci basterebbe fare `@drafting`.

Pensate possa tornare utile?

**Comment by Cadienvan** [comment]
Ciao Simone, assolutamente sì! L'abbiamo già fatto per il tech group e avrebbe senso per qualsiasi gruppo debba intervenire in qualsiasi forma.

Me lo prendo come punto e se non ci sono opinioni contrarie nei prossimi giorni procedo!

**Comment by Cadienvan** [comment]
Visto che alla fine abbiamo proceduto, chiudo!
